apps/cli-daemon/src/app/workspace/workspace.controller.ts

```diff
@@ -102,4 +115,24 @@ export class WorkspaceController {
 
     return target;
   }
+
+  private ngNewArgsFromDto(dto: WorkspaceCreateDto): string[] {
```

Comment:
this function should be in the service not in the controller

Reply:
It didn't feel right to have the service know anything about the DTO, and also this is the pattern that was used in component.controller.ts

@agroupp what do you say, does this belong in the controller or service?